manifests/generated/virt-handler.yaml.in

```diff
@@ -66,6 +66,7 @@ spec:
           name: virt-private-dir
         - mountPath: /var/lib/kubelet/device-plugins
           name: device-plugin
+      hostNetwork: true
```

Comment:
@rmohr I admit I don't fully understand why this should affect VNC / console, but as a data point, all tests, including console_test.go suite, pass with the changes.

Could you also be specific about port collisions? Perhaps a scenario so that I better grasp what you mean? (Sorry, never looked into this part of the system so I may need some hand holding here.)

As to why it's needed, it's because the handler need access to the root namespace to switch to namespaces of launcher pods. If handler is inside its own namespace, it won't see other namespaces there.

Reply:
Hm, we are in the hostPID namespace. As such you should see all namespace in /proc/<pid>/ns. If you want to look at the root network namespace you can look at /proc/1/ns. I think you just have to first switch to the pid1 namespace first.

As to port collision, we own all ports in our namespace. We can just blindly expose port 443 and know that no other service uses that in our virt-handler namespace. If we are in the root network namespace there can be other services which want that port too.